src/data/markdown/docs/30 xk6-browser/01 Getting started/01 Welcome.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Welcome'
+excerpt: 'xk6-browser brings browser automation and end-to-end testing to k6 while supporting core k6 features. Interact with real browsers and collect frontend metrics as part of your k6 tests.'
+---
+
+## What is xk6-browser?
+
+[xk6-browser](https://github.com/grafana/xk6-browser) brings browser automation and end-to-end web testing to k6 while supporting core k6 features. It adds browser-level APIs to interact with browsers and collect frontend performance metrics as part of your k6 tests.
+
+xk6-browser aims to provide rough compatibility with the Playwright API, so you don’t need to learn a completely new API.
+
+## Why xk6-browser?
+
+The [performance golden rule](https://www.stevesouders.com/blog/2012/02/10/the-performance-golden-rule/) states that 80%-90% of the website response time is spent on the frontend. But when the number of concurrent users is high, the percentage of time spent between the frontend and backend changes:
+
+![Frontend time vs Backend time with increasing amount of website visitors](./images/Frontend-Backend-LoadTesting.png)
```

Comment:
The image isn't displaying for me in the welcome page. Not sure why though :/
<img width="403" alt="Screenshot 2022-10-20 at 16 02 48" src="https://user-images.githubusercontent.com/1112428/196986332-8b5bdafa-334e-480f-bec8-bcffb77e523e.png">

Reply:
Yeah, in localhost it doesn't get displayed :( But if I deploy to Staging, it's displayed 🤷🏻‍♀️

https://mdr-ci.staging.k6.io/docs/refs/pull/828/merge/javascript-api/xk6-browser/getting-started/welcome/

This is also the same image used in https://mdr-ci.staging.k6.io/docs/refs/pull/828/merge/testing-guides/load-testing-websites/ and it also doesn't display locally.